lib/messagebird_credits_check_plugin.rb

```
@@ -0,0 +1,21 @@
+class MessagebirdCreditsCheckPlugin < Scout::Plugin
+  needs 'messagebird'
+
+  OPTIONS = <<-END
+    messagebird_api_token:
+      name: MessageBird API token
+  END
+
+  def build_report
+    return unless on_correct_util_server?
+
+    amount = MessageBird::Client.new(option(:messagebird_api_token)).balance.amount
+    report(balance: amount)
+  end
+
+  # Run this plugin on one util server, not both.
+  def on_correct_util_server?
+    File.exist?('/var/www/production.rom.roqua.nl/current/config/cron_state') &&
+      File.read('/var/www/production.rom.roqua.nl/current/config/cron_state').strip == 'on'
```

Dit is zoals Herman het graag heeft om te bepalen of er cronjobs worden gedraaid op de server. Het lijkt me eigenlijk net zo fragiel/robuust als het af laten hangen van een envvar.

Silent failure: scout geeft aan dat de plugin niet (goed) heeft gedraaid als het bestand niet is gevonden. Dus helemaal silent is het niet. Daar gaat een envvar ook niks aan veranderen.